src/components/com_tjcertificate/administrator/languages/en-GB/en-GB.com_tjcertificate.sys.ini

```diff
@@ -29,3 +29,6 @@ COM_TJCERTIFICATE_CERTIFICATE_LIST_DEFAULT_TITLE="Certificates"
 COM_TJCERTIFICATE_CERTIFICATE_LIST_DEFAULT_DESC="List of certificates"
 COM_TJCERTIFICATE_CERTIFICATE_FIELDSET_CLIENT_SELECT_LABEL="Client"
 COM_TJCERTIFICATE_CERTIFICATE_FIELDSET_CLIENT_SELECT_DESC="Client"
+
+;Add record menu
+COM_TJCERTIFICATE_ADD_EXTERNAL_CERTIFICATE="Add record"
```

Comment:
I think now you can change here too. Change `Add record` => `Add training record`

Reply:
done